02-lsp_extension/02.00-lsp_intro.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
<p> </p><p>"Restart JupyterNotebook for the changes to take effect and verify the installation." -&gt; Restart the Jupyter notebook or all of JupyterLab? Not clear to me.</p><p> </p><p>"If you click on it, you will see that it is either running or not." -&gt; An image might be nice to a user who doesn't know what to look for.</p><p> </p><p>"You may need to install additional dependencies to run the LSP server through the command line in terminal" -&gt; okay! So when I walked through this, I got the same thing. The message said that "conda: <code>conda install -c conda-forge jedi-language-server" was needed. </code>But instead of installing it after we already have our jupyterlab server running, can we do something to prevent this step for future users altogether? In other words, where else could we list our projects dependency on jedi-language-server? A hint, you may find it useful to run the command <code>conda env update --file binder/environment.yml&nbsp;--prune</code> at some point. Unfortunately, I tried to fix this problem on my machine and got unsolvable conflicts when I tried to add jedi-language-server to the conda environment. I'm wondering how you were able to successfully get this working on your end. Please make an issue logging what steps you remember taking to get the lsp server working. We will probably have to meet about this one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nicole-brewer/jupyterlab-ide/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>